system/settings/settings.xml

```diff
@@ -2544,7 +2548,7 @@
           <level>2</level>
           <default>false</default>
           <dependencies>
-            <dependency type="visible">
+            <dependency type="enable">
```

Comment:
greying out a setting that can't ever be activated on a system because the hw does not support it makes no sense.